src/shared/logs-show.c

```diff
@@ -317,15 +317,46 @@ static bool print_multiline(
         return ellipsized;
 }
 
-static int output_timestamp_monotonic(FILE *f, const dual_timestamp *ts) {
+static int output_timestamp_monotonic(
+                FILE *f, OutputMode mode,
+                const dual_timestamp *ts,
+                const sd_id128_t *boot_id,
+                const dual_timestamp *previous_ts,
+                const sd_id128_t *previous_boot_id) {
+        int written_chars = 0;
```

Comment:
Please insert an empty line between function arguments and variable declaration(s).

Reply:
Done